src/main/java/carRacing/domain/Race.java

```diff
@@ -0,0 +1,37 @@
+package carRacing.domain;
+
+import java.util.List;
+
+import static carRacing.domain.RandomNumberGenerator.getRandomNumber;
+
+public class Race {
+    private static final int MOVE_CONDITION = 4;
+    private static final boolean MOVE_SUCCESS = true;
+    private static final boolean MOVE_FAIL = false;
+    private static final int MOVE_DISTANCE = 1;
+
+
+    public static List<Car> moveCars(List<Car> cars) {
+        for (Car car : cars) {
+            int randomNumber = getRandomNumber();
+            if (moveStatus(randomNumber)) {
+                moveCar(car);
+            }
+            int currentLocation = car.getLocation();
+
+        }
+        return cars;
+    }
+
+    public static void moveCar(Car car) {
+        int currentLocation = car.getLocation();
+        car.setLocation(currentLocation + MOVE_DISTANCE);
+    }
```

Comment:
자동차의 위치 값을 getter로 꺼내어서 setter를 활용해 할당해 주셨네요.
자동차가 이동하는 행위는 핵심 로직을 보여지는데요. setter 보단 메시지를 보내어 자동차의 위치를 움직이게 하면 어떨까요?

```suggestion
    public static void moveCar(Car car) {
        car.move();
    }
```